Lib/test/test_socket.py

```diff
@@ -4705,10 +4705,14 @@ def testSetBlocking_overflow(self):
             self.skipTest('needs UINT_MAX < ULONG_MAX')
 
         self.serv.setblocking(False)
+        if fcntl:
+            self.assertFalse(_is_fd_in_blocking_mode(self.serv))
```

Comment:
I don't see _is_fd_in_blocking_mode() in the main branch. I would prefer to have the same tests in 3.11 and main.

Moreover, I don't see where _is_fd_in_blocking_mode() is defined.

Reply:
It was removed in 304315d251dbb4e85dd86056ba1925f25e646ca1. I think that `assert_sock_timeout()` can be used instead.